### PR TITLE
Seethrough component for shipping containers, big bones, ratvar wreck and necropolis gate

### PR DESCRIPTION
**Pull request by Time-Green**
Big sprite, easy conversion. Let's you see through a bunch of stuff with big sprites

:cl:
qol: you can look behind shipping containers, ratvars wreck, through the necropolis gate and behind the big bones
/:cl:

Behind a container

https://user-images.githubusercontent.com/7501474/201395747-4843b2df-d3ad-4047-b4af-66310d1d0472.mp4

Through the necropolis gate

https://user-images.githubusercontent.com/7501474/201416066-9f69399d-f757-4132-941d-5cf3b98ced5d.mp4

### CODE REVIEW

**Comment by ATH1909** [comment]
holy shit

**Comment by axietheaxolotl** [comment]
image
ayo dude

**Comment by Time-Green** [comment]
> image
> ayo dude

I keep mixing up pull and merge, okay. But this is a pretty obvious explanation of what not to do

File: code/__HELPERS/see_through_maps.dm [hunk 1]
```diff
@@ -10,6 +10,10 @@
 #define SEE_THROUGH_MAP_THREE_X_TWO "3x2"
 ///One row of three tiles above the atom, but offset one tile to the left because of how billboards work
 #define SEE_THROUGH_MAP_BILLBOARD "billboard"
+///Two rows of three wide, but offset one tile to the left because thats how shipping containers work
+#define SEE_THROUGH_MAP_SHIPPING_CONTAINER "shipping_container"
+///I'm not even gonna try and describe it
```
Comment (Mothblocks) [inline code/__HELPERS/see_through_maps.dm:15]: You do have to document this, you don't need to describe exactly what it is you just need to describe where it's used

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Pull request merged**